### DISCUSSION TITLE
Avoiding whitelisting for customer domains

### DISCUSSION THREAD

**Discussion by theDurin** [phantom/docs#281] [Help & Support] [answered]
### Summary

I have a Solana payment widget that gets flagged by blowfish.xyz on non-whitelisted domains. I want to have my widget on multiple domains and the easiest way I have found to get around the domain restriction is to embed it via iframe. But this limits the wallet adapter, since not all wallets are injecting themselves in child frames (Phantom doesn't, for example), so users can't connect their wallet.

Is it possible to manually inject Phantom into the iframe? Or is there some other way to handle this without messaging blowfish for each new customer? I haven't found any useful documentation for that.
How would this be implemented?

### Steps to Reproduce

-

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hey @theDurin,

I responded to your support email as well. But just to reiterate I think you're only options are to not use iFrames, since we the provider is not available in iFrames and to email blowfish to see if they can accommodate your domain creation plan in some way.
